src/bidiMapper/BidiNoOpParser.ts

```diff
@@ -56,6 +56,11 @@ export class BidiNoOpParser implements IBidiParser {
   ): BrowsingContext.SetViewportParameters {
     return params as BrowsingContext.SetViewportParameters;
   }
+  parseHandleUserPromptParams(
+    params: object
+  ): BrowsingContext.HandleUserPromptParameters {
+    return params as BrowsingContext.HandleUserPromptParameters;
+  }
 
```

Comment:
Note that all commands within each Bidi*Parser.ts are sorted alphabetically within each domain, it would be nice to keep doing so

Reply:
Please when introducing such changes use the `keep-sorted` it has a method `// keep-sorted start block=yes`
that will work for JS methods

Reply:
I did not realize keep-sorted worked with multiple lines. That's nice!

Could you make this change in this PR then?

Reply:
Done it for you